### ISSUE TITLE
What to do with these fallback types(CompanyParams, PersonParams) ?

### ISSUE THREAD

**Issue by marcoslopes** [arlyon/async-stripe#49]
Hi first of all thank you very much for the stewardship of this library,
Quick question, trying to use the connect features I can see that [these types](https://github.com/arlyon/async-stripe/blob/e397913f7a4dd43f06abc97e2aff0a80563aaaa0/src/resources.rs#L237-L249) are overriding [actual](https://github.com/arlyon/async-stripe/blob/master/src/resources/generated/account.rs#L938) [useful](https://github.com/arlyon/async-stripe/blob/master/src/resources/generated/account.rs#L1082) generated types, that allows the Account creation call to be made.
Not sure if that was left as a placeholder for the future in which case:

-  I would change the feature from "account" to "connect" as "account" feature is [not present anymore](https://github.com/arlyon/async-stripe/blob/master/Cargo.toml#L35);
-  or if it just needs to be removed --this is what i would probably do-- but i don't understand the library in depth so just requesting your advice.

Thanks again

**Comment by arlyon** [comment]
Hi! Thanks for bringing this to my attention. These stubs are there because they are needed in `token.rs`, and may not always be there if you don't have the right feature flag, _however_,

- the feature is named incorrectly (good catch!)
- the types may need updating and _at the least_ some documentation explaining why they are there

From a quick check, seems that when you create an [account token](https://stripe.com/docs/api/tokens/create_account) you can pass in either `PersonParams` or `CompanyParams` but I suspect that these were placed here just to get it to compile as it only supports `metadata`.

So we can either tweak the codegen to generate these types elsewhere than `account.rs` so they can be used independent of the `connect` flag, or just leave the stubs as they are, update their feature flag, and put a handy doc comment in to explain the logic and help the next guy.

**Comment by marcoslopes** [comment]
Awesome @arlyon, thanks for the prompt reply.

I tried having a look into the codegen bit but I failed to see where I could easily move the `#/components/schemas/legal_entity_company` from `account.rs` to a more appropriate place, and the respective `#/components/schemas/person` as well.
There is a `types.rs` but that seems to be manually generated, which I don't believe would be ideal, I also didn't see a place to exclude them anyways(maybe the codegen is smart enough to know).
I'll try and dig deeper again tomorrow and if I still fail to see the connection, I'll submit a pr with the second suggested approach if that is alright.

Thanks again.
